tasks/extract_filepaths_for_granule/schemas/input.json

```diff
@@ -0,0 +1,244 @@
+      },
+      "required": [
+        "granules"
+      ]
+    },
+    "config": {
+      "type": "object",
+      "properties": {
+        "glacier-bucket": {
+          "type": "string"
+        },
+        "protected-bucket": {
+          "type": "string"
+        },
+        "internal-bucket": {
+          "type": "string"
+        },
+        "private-bucket": {
+          "type": "string"
+        },
+        "public-bucket": {
+          "type": "string"
+        },
+        "file-buckets": {
+          "type": "array",
+          "items": [
+            {
+              "type": "object",
+              "properties": {
+                "regex": {
+                  "type": "string"
+                },
+                "sampleFileName": {
+                  "type": "string"
+                },
+                "bucket": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "regex",
+                "sampleFileName",
+                "bucket"
+              ]
+            },
+            {
+              "type": "object",
+              "properties": {
+                "regex": {
+                  "type": "string"
+                },
+                "sampleFileName": {
+                  "type": "string"
+                },
+                "bucket": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "regex",
+                "sampleFileName",
+                "bucket"
+              ]
+            },
+            {
+              "type": "object",
+              "properties": {
+                "regex": {
+                  "type": "string"
+                },
+                "sampleFileName": {
+                  "type": "string"
+                },
+                "bucket": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "regex",
+                "sampleFileName",
+                "bucket"
+              ]
+            },
+            {
+              "type": "object",
+              "properties": {
+                "regex": {
+                  "type": "string"
+                },
+                "sampleFileName": {
+                  "type": "string"
+                },
+                "bucket": {
+                  "type": "string"
+                }
+              },
+              "required": [
+                "regex",
+                "sampleFileName",
+                "bucket"
+              ]
+            }
+          ]
+        },
+        "excludeFileTypes": {
+          "type": "array",
+          "items": [
+            {
+              "type": "string"
+            }
+          ]
+        }
+      },
+      "required": [
+        "glacier-bucket",
+        "protected-bucket",
+        "internal-bucket",
+        "private-bucket",
+        "public-bucket",
+        "file-buckets",
+        "excludeFileTypes"
```

Comment:
I don't believe `excludeFileTypes` is required as this is an optional configuration item. @AndrewDorn please make sure what I am saying is correct as a user may or may not have this value configured for a collection so it is possible that this key in configuration may also not exist or be None

Reply:
Yup. The way the code is written, excludeFileTypes will default to [] if not present, so it doesn't need to be required.

Reply:
Presently protected, internal, private, public, and file values are required, but I understand that section is presently in flux.